src/devices/tpms_ford.c

```diff
@@ -52,12 +57,21 @@ static int tpms_ford_decode(r_device *decoder, bitbuffer_t *bitbuffer, unsigned
 
     code = b[4]<<16 | b[5]<<8 | b[6];
     sprintf(code_str, "%06x", code);
+    
+    pressure = 0.3 + b[4] * 0.25;
+    pressure_bar = pressure * 0.0689475729;
+    sprintf(temperature_str, "%.2f%s%.2f%s", pressure, "psi ~", pressure_bar, "bar");
+    
+    temperature = b[5]-64;
+    sprintf(temperature_str, "%d%s", temperature, "°C");
 
     data = data_make(
         "model",        "",     DATA_STRING, "Ford",
         "type",         "",     DATA_STRING, "TPMS",
         "id",           "",     DATA_STRING, id_str,
         "code",         "",     DATA_STRING, code_str,
+        "pressure",         "",     DATA_STRING, pressure_str,
+        "temperature",         "",     DATA_STRING, temperature_str,
```

Comment:
make this `"temperature_C", "", DATA_INT, temperature,` please. remove "temperature_str"